applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -78,6 +78,39 @@ def test_Naca0012SmallCompressible(self):
             for file_name in os.listdir():
                 if file_name.endswith(".time"):
                     kratos_utilities.DeleteFileIfExisting(file_name)
```

Comment:
You can try putting this line out of the scope of the "with WorkFolderScope(work_folder):"

```suggestion
        kratos_utilities.DeleteTimeFiles(work_folder)
```

(not sure whether this suggested change tool puts the right indentation)

Reply:
That works!